### PR TITLE
add cmake command to make templates for elevator demo (reopen)

### PR DESCRIPTION
**Pull request by furushchev**
related to https://github.com/jsk-ros-pkg/jsk_recognition/pull/702

### CODE REVIEW

File: jsk_perception/CMakeLists.txt [hunk 1]
```diff
@@ -142,11 +142,11 @@ add_dependencies(rectangle_detector       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}
 add_dependencies(color_histogram_sliding_matcher       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
 
 
-#add_custom_command(
-#  OUTPUT  ${PROJECT_SOURCE_DIR}/template
-#  DEPENDS ${PROJECT_SOURCE_DIR}/src/eusmodel_template_gen.l
-#  COMMAND ${PROJECT_SOURCE_DIR}/src/eusmodel_template_gen.sh)
-#add_custom_target(eusmodel_template ALL DEPENDS ${PROJECT_SOURCE_DIR}/template)
+add_custom_command(
+ OUTPUT  ${PROJECT_SOURCE_DIR}/template
```
Comment (garaemon) [inline jsk_perception/CMakeLists.txt]: Can you generate under devel/share directory? I don't want to contaminate src directory

**Comment by furushchev** [comment]
moved to devel/share/jsk_perception/template

**Comment by furushchev** [comment]
is that directory accessable easily from euslisp?

**Comment by garaemon** [comment]
Please update `load` function of euslisp to access devel directory

**Comment by furushchev** [comment]
you mean we need to add the feature to access to devel directory to euslisp?

**Comment by garaemon** [comment]
> you mean we need to add the feature to access to devel directory to euslisp?

Yes

For example, update load function to check directories which is available by `catkin_find`
```
$ catkin_find jsk_pcl_ros
/home/leus/ros/hydro/devel/include/jsk_pcl_ros
/home/leus/ros/hydro/devel/lib/jsk_pcl_ros
/home/leus/ros/hydro/devel/share/jsk_pcl_ros
/home/leus/ros/hydro/src/jsk-ros-pkg/jsk_recognition/jsk_pcl_ros
```

**Comment by garaemon** [comment]
Another idea is to have special directory for euslisp like python such as `lib/euslisp/` and add the directory to `*load-path*` and use require.

**Comment by furushchev** [comment]
ok. I do the first tonight. Thanks!

**Comment by garaemon** [comment]
```
[jsk_perception] /workspace/ros/ws_jsk_recognition/src/jsk_recognition/jsk_perception/src/eusmodel_template_gen.sh: line 26: ./src/eusmodel_template_gen.l: Permission denied 
```

**Comment by furushchev** [comment]
is this because `roseus` binary not found?
should we need to use bad know how like used in `jsk_maps`?(writing lisp code in CMakeLists.txt?)

**Comment by garaemon** [comment]
It's not executable
```
$ ls -l eusmodel_template_gen.l
-rw-r--r-- 1 garaemon garaemon 1060 Feb 11 20:20 eusmodel_template_gen.l
```

**Pull request closed without merging**